### DISCUSSION TITLE
Button to toggle alert rules status

### DISCUSSION THREAD

**Discussion by jufy111** [grafana/grafana#67721] [Alerting]
Hi,

I'd love the ability to quickly enable/disable the alert rules in the Web-UI without having to create a silence each time. 
It would make sense to have a toggle button under the actions (see image below).

![image](https://user-images.githubusercontent.com/40817638/235910417-334c9614-73f6-465f-8613-3432f6a95b45.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82669. Feel free to continue the discussion around this there. Thank you!
